docs/orchestrate/best-practices.md

```diff
@@ -10,11 +10,11 @@ This page describes best practices for using container images in Sauce Orchestra
 
 ## Building Efficient Java Images
 
-Java is one of the most popular programming languages available and is widely used for Selenium and Appium test projects. Here are some tips and tricks to get your Java project working with Sauce Orchestrate.
+Java is one of the most popular programming languages available and is widely used for Selenium and Appium test projects. Here are some tips and tricks to get your Java prjoect working with Sauce Orchestrate. For this documentation page we will assume that Docker is being used.
```

Comment:
Looks like this PR re-introduces issues that were previously fixed in PR https://github.com/saucelabs/sauce-docs/pull/1949 ~ @mikedon Is the branch for this PR based off of the main branch?

```suggestion
Java is one of the most popular programming languages available and is widely used for Selenium and Appium test projects. Here are some tips and tricks to get your Java project working with Sauce Orchestrate. For this documentation page we will assume that Docker is being used.
```